### ISSUE TITLE
Remove Drupal user images from Drupal.org profiles

### ISSUE THREAD

**Issue by frankmuse** [drupaldiversity/administration#49]
Hi again! I apologize for all the posts. It's just a few, I promise! I've had a couple ideas that I've wanted to share for a while and finally I'm INSPIRED to just post them all now!

This idea involves removing Drupal user images. The reason is that showing an image of users on the site may lead to discrimination based on conscious or subconscious prejudices when community members interact. It's that simple. I don't think I need to delineate all the ways a picture of a person might lead to discrimination.

**Comment by stephenpurkiss** [comment]
> I don't think I need to delineate all the ways a picture of a person might lead to discrimination.

A username could also lead to discrimination.

For all the positives user images bring, being able to express your individuality or whatever, removing them in order to deal with "conscious or subconscious prejudices" is a stretch too far IMHO.

We could just display a number, or a random ID each post, but after a while you could get to know who's who.

And of course you don't have to post an actual picture of yourself, every time I look at my current profile pic I think OMG I'm so fat right now there's my manboobs on display, so I'm kinda discriminating against myself. But that's my problem.

So I'm not for this, personally I find images help. That of course is only my view, but I saw this fly by and thought I'd reply.

**Comment by frankmuse** [comment]
Thanks! I see your point. People could put their real names as their usernames, which could also lead to prejudice.

Maybe we should just add a big disclaimer to both images and user names to PLEASE DON'T INCLUDE YOUR REAL IMAGE OR NAME! Maybe even moderation to prevent them from doing so? Hear me out, even if they don't belong to some disadvantaged group, their picture may trigger unconscious bias or fear from other community members.

**Comment by stephenpurkiss** [comment]
Maybe we should treat people as adults and let them do whatever they want to do with regards to setting their own usernames and profile pics.

You can't control how other people think. If they act in inappropriate ways as set out by the newly adopted open source contributor code of conduct then sure, deal with it appropriately but when we get into dealing with triggering unconscious bias or fear let's leave that up to the marketers of the world.

**Comment by stephenpurkiss** [comment]
Saw the guy who wrote this book on @garyvee show the other day, I like this way of thinking about things:

Black Privilege: Opportunity Comes to Those Who Create It 
https://www.amazon.co.uk/d/Books/Untitled-Charlamagne-Tha-God/1501145304

It's about not wanting to tear down other people's stuff but instead we all have the ability to build our own.

I don't believe sanitised environments lead to more opportunity, I believe things like avatars help people to connect. I'm making my 'statement' with my Free Software Free Society t-shirt, I like that. I don't want that to be taken away, and it sucks being "white privileged male" cos that will no doubt be the thing that's thrown at me if I say anything about it. Like now.

**Comment by frankmuse** [comment]
Thanks, I will look at that book.

I know you don't mean it, but I have to say though that, "treat people as adults and let them do whatever they want to do..." is the number 1 excuse given to just enable/acquiesce to discriminatory behavior.

Sure, it's easy to be "treated as an adult" if you are a cisgender white male, but consider if people would look at your image and then not accept your commit? Or disagree with your comment? Or what if someone posts a confederate flag for their user image? Or a picture of Hitler? What if I'm a new member to the community and I have to see that? Will I feel included?

Maybe images should be allowed... but maybe we could have a pre-selected list of images that they can choose from so they can "express their individuality"? Or at least maybe we can enforce strict moderation on custom images.

**Comment by stephenpurkiss** [comment]
Well that's exactly what I do mean by treat people as adults. A few years back I built the what-they-call-headless back-end for Nickelodeon Jr. "Create" where kids paint over line art and they can send it to friends but they're only allowed to use certain words. Or when we built toonix.com and you select body parts etc. from a limited set. There's strict rules about what you can and can't allow kids to do so they have to be very careful. Of course they can't advertise to kids either so instead they just spend millions on these games instead to get them all hooked on their brands. So evil happens and the more you suppress it the more it finds ways around it.

So when I say treat people as adults, what I mean is let's encourage the freedom to join the community and not restrict the shit out of everything as I truly believe that only puts people off further. 

I would much rather focus on improving communication tools so it's not just all text. The more people connect and bond the closer relationships get, that's why events IRL have always been such a big part of Drupal. 

When I first posted to the community I'd just written a block to display my LinkedIn profile. I'd requested CVS access but not heard back so I posted to the forums with a link and got no reply. Well I did 6 years later but only after I mentioned it at BadCamp ;)

Please, let's focus on things like welcoming people, not restricting and homogenising. We can always chuck assholes out if they do bad things, but to pre-empt is also way badder.

**Comment by frankmuse** [comment]
I just want to protect people rather than fix things AFTER they happen.

**Comment by stephenpurkiss** [comment]
Focus on things you can control, and controlling other people's minds isn't the best way forward for humanity IMHO.

You can't control how others react, only how you react to something.

Be more welcoming, show a good environment. For example, have a look at a wine bar or coffee shop or restaurant - they all have their own atmosphere and the clientelle gravitate to what suits them. Create an atmosphere that you want.

Doing this for everyone in the world is going to be an impossible task. I've avocated decentralisation as I don't think it's possible to build one place that fits all, so the only other option is to offer options.

**Comment by stephenpurkiss** [comment]
I'm reminded of an experiment where they took the traffic signs & lights out from somewhere and people actually drove more carefully.

Removing pictures from profiles would go against what every other community does, and the reasoning is something which is not under our control. We're here to collaborate on making the best tool for people of all abilities to be able to make use of this wonderful web, whether they are able to code or not. We're not here to teach people how to behave, but we do need to set guidelines which is what I believe this group effort is doing and seems to be doing it pretty well from what I've seen over the past few days, although I'm not on the chat so miss whatever goes on there.

Drupal has taught me a lot about people from other walks of life - I'd never met a transgender person before, I thought all Spanish were X, German were Y, etc. - just the stereotypes we get fed. But then the more I interacted with people the more I enjoyed learning about different cultures and ways of thinking. We should be encouraging more of that, as I said not just offline but also online. Text only doesn't do us any justice, you need VITAL comms - Video, Images, Text, Audio & Links.

Anyways, I get the feeling I'm talking too much and probably annoying someone so I'll stop now. G'nite!

**Comment by NonProfit-Hans** [comment]
Steve is correct here. Killing the avatar won't stop marginalization. Allowing people to express themselves and demonstrate we are all real people will only help. Eventually someone will cross the line. People will take offense, perhaps legitimately. So I'm in support of a policy which says "Be cool" and I'm OK with ejecting members who are out to reek havoc. However, that's not the type of individual Drupal typically attracts. My world has been expanded greatly by many wonderful people from this community. Please, let them be themselves.

**Comment by frankmuse** [comment]
Thanks for unilaterally declaring Steve's correctness, Hans (and, by doing so, marginalizing my opinion, just btw). I'm glad you think that it will "only help". If we have criteria for "ejecting members" which you're OK with, why not eject those members BEFORE they have a chance to cause harm in the community? 

I'm sorry, if you're not part of a disadvantaged group, there is some harm that you just can't see. Even if you are, there are discriminations that are too subtle for you to see. It might be difficult to determine which users are "reek[ing] havoc" if their influence is subconscious! Why not just prevent the problem in the first place?

Of course this doesn't "stop marginalization"! Does that mean we can't take steps to prevent marginalized groups from being taken advantage of, even if that means it's by subconscious bias?

**Comment by NonProfit-Hans** [comment]
Hi frankmuse, thanks for your reply.

I did unilaterally declare Steve's correctness. That's what's known as my opinion. How do you recommend we discuss ideas without implying some ideas have greater merit than others? I may well have marginalized your opinion, but I did not marginalize you. I did not assume you are not part of a disadvantaged group and thus unable to identify harm. I did not state there is discrimination too subtle for you to see. Nor did I imply there is discrimination you are simply not perceptible enough to notice. 

_Why not eject those members BEFORE they have a chance to cause harm in the community?_ Because we would very quickly have a community of one. Each of us have the opportunity to cause harm. Each of us are bothered by the actions of others. Drupal is not the perfect community because it is made up of individuals each of who are imperfect. Never-the-less, we've been able to come together and create some amazing software that one or ten (or possibly ten thousand) never could have. 

_Does that mean we can't take steps to prevent marginalized groups from being taken advantage of..._ Of course we should seek to prevent everyone from being offended when it includes any form of identifiable harm. But when I'm offended of a selfie you took of yourself while enjoying last holiday; that's my issue not Drupal's.

Your permission to speak should never be contingent upon my willingness to listen.

**Comment by NonProfit-Hans** [comment]
Hi frankmuse, thanks for your reply.

I did unilaterally declare Steve's correctness. That's what's known as my opinion. How do you recommend we discuss ideas without implying some ideas have greater merit than others? I may well have marginalized your opinion, but I did not marginalize you. I did not assume you are not part of a disadvantaged group and thus unable to identify harm and did not state there is discrimination you are simply not perceptible enough to notice.

_Why not eject those members BEFORE they have a chance to cause harm in the community?_ Because we would very quickly have a community of one. Each of us have the opportunity to cause harm. Each of us are sometimes bothered by the actions of others. Drupal is not the perfect community because it is made up of individuals each of who are imperfect. Never-the-less, we've been able to come together and create some amazing software that one or ten or ten thousand, never could have.

_Does that mean we can't take steps to prevent marginalized groups from being taken advantage of..._ Of course we should seek to prevent everyone from being targeted when it includes any form of identifiable harm. But when I'm offended of a selfie you took of yourself while enjoying last holiday; that's my issue not Drupal's.

Your permission to speak should never be contingent upon my willingness to listen.

**Comment by NonProfit-Hans** [comment]
Hi frankmuse, thanks for your reply.

I did unilaterally declare Steve's correctness. That's what's known as my opinion. How do you recommend we discuss ideas without implying some ideas have greater merit than others? I may well have marginalized your opinion, but I did not marginalize you. I did not assume you are not part of a disadvantaged group and thus unable to identify harm and did not state there is discrimination you are simply not perceptible enough to notice.

_Why not eject those members BEFORE they have a chance to cause harm in the community?_ Because we would very quickly have a community of one. Each of us have the opportunity to cause harm. Each of us are sometimes bothered by the actions of others. Drupal is not the perfect community because it is made up of individuals each of who are imperfect. Never-the-less, we've been able to come together and create some amazing software that one or ten or ten thousand, never could have.

_Does that mean we can't take steps to prevent marginalized groups from being taken advantage of, even if that means it's by subconscious bias?_ Of course we should seek to prevent everyone from being targeted when it includes any form of identifiable harm. But when I'm offended of a selfie you took while enjoying last holiday; that's my issue not Drupal's.

Your permission to speak should never be contingent upon my willingness to listen.

**Comment by NonProfit-Hans** [comment]
Hi frankmuse, thanks for your reply.

I did unilaterally declare Steve's correctness. That's what's known as my opinion. How do you recommend we discuss ideas without implying some ideas have greater merit than others? I may well have marginalized your opinion, but I did not marginalize you. I did not assume you are not part of a disadvantaged group and thus unable to identify harm and did not state there is discrimination you are simply not perceptible enough to notice.

_Why not eject those members BEFORE they have a chance to cause harm in the community?_ Because we would very quickly have a community of one. Each of us have the opportunity to cause harm. Each of us are sometimes bothered by the actions of others. Drupal is not the perfect community because it is made up of individuals each of who are imperfect. Never-the-less, we've been able to come together and create some amazing software that one or ten or ten thousand, never could have.

_Does that mean we can't take steps to prevent marginalized groups from being taken advantage of, even if that means it's by subconscious bias?_ Of course we should seek to prevent everyone from being targeted when it includes any form of identifiable harm. But when I'm offended by a selfie you took while enjoying last holiday; that's my issue not Drupal's.

Your permission to speak should never be contingent upon my willingness to listen.

**Comment by rachellawson** [comment]
I'm on mobile so struggling to find the Drupal.org issue right now but profile images were introduced on issues precisely to subconsciously remind us that we are all humans when interacting with each other. 
Having the images there has been a benefit and I'd hate to see them removed.

**Comment by david-hernandez** [comment]
Just confirming what Rachel said. The profile images were added a couple years, after much discussion, to be seen as largely a positive move to humanization. But, there is nothing mandatory about them or the username. No one dictates your username, and you don't have to use a photo at all. You can pretty much make your profile about as revealing or anonymous as you'd like.

**Comment by YesCT** [comment]
Please see https://www.drupal.org/node/2291555
Change wording from User Picture to Icon or Avatar

which has some steps forward, and also related history of issues.

**Comment by damienmckenna** [comment]
Everyone in this conversation could do with looking over this study, it's completely relevant: https://peerj.com/preprints/1733/?td=sd

**Comment by NonProfit-Hans** [comment]
Hi Damien, thanks for the link.

I did not read the full publication, but if I understand it correctly, a study was conduced which considered GitHub contributions. The research showed that women's contributions tend to be accepted more often than men's but only when the contributor is not identifiable as women. My hope is this type of behavior is, or one day will be, completely absent from Drupal.

In a virtual community, it's certainly possible to eradicate references to gender. There are many indicators beyond the avatar, though. This would certainly include the gender field on the user profile and potentially user names and links to third-party sites which includes personal, professional and social media. Most importantly, we could never gather together in person. However, even if we were able to completely obscure the gender identity of every individual, the net result would be that the users who get it would continue to get it and those who don't would assume every contribution of merit was _naturally_ made by a middle-class white guy.

The alternative is to celebrate diversity. It is to read a blog and realize, "I've never thought of that, they're absolutely right!" It is to review code and say, "This is amazing, who wrote it?" It is to exit a session stating, "That was the best one I attended!" It is to work along side someone who is somehow different and realize "The project would have been at a loss had they not been participating."

The solution to discrimination is never to force people within marginalized groups to hide their identity.

**Comment by fuzzbomb** [comment]
> why not eject those members BEFORE they have a chance to cause harm in the community?

Because they haven't caused any harm, and this would deny them the chance to do good in the community.

**Comment by sugaroverflow** [comment]
I am closing this issue because I feel that it's out of scope - and we are not Drupal.org. 
Thank you!

Quoting @YesCT's comment earlier: 
> Please see https://www.drupal.org/node/2291555
Change wording from User Picture to Icon or Avatar
